views_api.py

```diff
@@ -270,3 +271,16 @@ async def api_check_fiat_rate(currency):
         rate = None
 
     return {"rate": rate}
+
+
+## ITEMS
+@tpos_ext.put("/api/v1/tposs/{tpos_id}/items", status_code=HTTPStatus.CREATED)
```

Comment:
- this endpoint is fine if more items have to be added at once (importing from a file for example)
- an endpoint should exist for adding items one by one
  - it gets the existing tpos with its items
  - adds an item to the list
  - persists the updated list

- same for `delete`

Reply:
The items is just a string, there's no id, i can't pull just one! it just replaces the entire string on the DB.

Reply:
If there is an UI bug (eg: sends an empty array instead of the full list) then the entire items list is lost (this can mean hundreds of items).

Also if 3rd party systems would like to add items via the API then they would first have to fetch all existing items, then update. This might not be acceptable.

Reply:
I prefer not to deal with that in this PR, but rather we can improve the items API in subsequent PR(s).